lambdas/process-journey-step/src/main/resources/statemachine/ipv-core-refactor-journey.yaml

```diff
@@ -79,6 +79,13 @@ IPV_IDENTITY_START_PAGE:
       response:
         type: journey
         journeyStepId: /journey/check-existing-identity
+    end:
+      type: basic
+      name: end
+      targetState: CHECK_EXISTING_IDENTITY
```

Comment:
I think we need this to be a new page state eg. `IPV_IDENTITY_POSTOFFICE_START` ?